proposal.rst

```diff
@@ -316,4 +445,4 @@ you plan to use, please give details.**
 .. to clearly identify that code (and keep existing licensing and copyright
 .. details intact), and to check with the mentors that it is OK to use.
 
-FILLME
+I would have to use external libraries. These libraries will belong to free software and be under licenses compatible with MIT/X license and GNU GPL version 2 and later.
```

Comment:
Are all of them compatible with both GPLv2+ and MIT/X?

Reply:
No, the libraries that I have proposed belong to the Mozilla Public License (MPL) version 2.0.
I have updated the PR.